Doc/whatsnew/3.14.rst

```diff
@@ -92,6 +92,12 @@ ast
 Added :func:`ast.compare` for comparing two ASTs.
 (Contributed by Batuhan Taskaya and Jeremy Hylton in :issue:`15987`.)
 
+asyncio
+-------
+
+Made :class:`asyncio.Queue` an :term:`asynchronous iterable`.
+(Contributed by Wannes Boeykens in :gh:`120491`.)
```

Comment:
You need to put the github issue and not the github PR (so it's 119154 and not 120491).

Reply:
```suggestion
(Contributed by Wannes Boeykens in :gh:`119154`.)
```